src/main/java/org/apache/commons/io/filefilter/RegexFileFilter.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.commons.io.filefilter;
 
+import org.apache.commons.io.IOCase;
```

Comment:
according to minimal change, this import's position change is unnecessary to this issue, suggest revert these 4 lines changes.